apps/test/unit/templates/teacherDashboard/AddSectionDialogTest.jsx

```diff
@@ -85,36 +85,14 @@ describe('AddSectionDialog', () => {
     expect(wrapper.find('Connect(EditSectionForm)').length).to.equal(0);
   });
 
-  it('once login type and audience are set EditSectionForm shows', () => {
```

Comment:
out of curiosity, why delete this test?

Reply:
I was realizing that once the flag and admin requirement are gone, it would depend on what the userId is which flow they went to. Now that I think about it though, I'd rather have a bunch of tests to adjust with ids during launch that fail correctly than the speed of getting to bypass them

Reply:
Added it back in!